src/components/views/auth/CaptchaForm.js

```diff
@@ -62,7 +62,7 @@ export default createReactClass({
             console.log("Loading recaptcha script...");
             window.mx_on_recaptcha_loaded = () => {this._onCaptchaLoaded();};
             let protocol = global.location.protocol;
-            if (protocol === "vector:") {
+            if (["vector:", "chrome-extension:", "moz-extension:"].includes(protocol)) {
```

Comment:
I'm wondering if it'd be simpler to just do `protocol !== "http:"` is there any downside to this which I am not seeing?

Reply:
AFAIK it's generally not possible to `fetch` from custom protocols - so personally I think that would work; tho I'm not too familiar with the code base yet, so dunno whether it might have side-effects.

Reply:
We should probably protect against the people who host on ftp and stuff. Keeping the whitelist feels sane.

Reply:
If you're somehow serving the site over ftp then your recaptcha will be inaccessible at that's only available over http/s

Reply:
I am on board with @t3chguy's suggestion... Forcing towards https for anything other than http seems safe. I don't see a way it would be security risk. If someone does, please shout!

@stoically, want to update this as @t3chguy suggests?

Reply:
Sounds good, thanks! Updated accordingly.